examples/upsert-resource-at-path/.expected/exec.sh

```diff
@@ -0,0 +1,4 @@
+#! /bin/bash
+
+# shellcheck disable=SC2016
+kpt fn eval -i upsert-resource:unstable --image-pull-policy never --fn-config .fn-config/fn-config.yaml
```

Comment:
Our test harness supports eval and fn-config, why not using that?
Ref: https://github.com/GoogleContainerTools/kpt/tree/main/pkg/test/runner
```yaml
testType: eval
image: ...
fnConfig: ...
```

Reply:
Updated.